src/Sylius/Bundle/CoreBundle/Doctrine/ORM/OrderRepository.php

```diff
@@ -311,11 +311,11 @@ public function findOrdersUnpaidSince(\DateTimeInterface $terminalDate): array
     {
         return $this->createQueryBuilder('o')
             ->where('o.checkoutState = :checkoutState')
-            ->andWhere('o.paymentState != :paymentState')
+            ->andWhere('o.paymentState = :paymentState')
             ->andWhere('o.state = :orderState')
             ->andWhere('o.checkoutCompletedAt < :terminalDate')
             ->setParameter('checkoutState', OrderCheckoutStates::STATE_COMPLETED)
-            ->setParameter('paymentState', OrderPaymentStates::STATE_PAID)
+            ->setParameter('paymentState', OrderPaymentStates::STATE_AWAITING_PAYMENT)
```

Comment:
What about different payment states, did you think about `partially_authorized` or `authorized`?